Sources/Sentry/SentryUIViewControllerSwizziling.m

```diff
@@ -13,6 +13,14 @@
 #if SENTRY_HAS_UIKIT
 #    import <UIKit/UIKit.h>
 
+/**
+ * UIApplication must conform to SentryUIApplication
+ * in order to swizzle work.
```

Comment:
`l`: I think this is not a proper sentence. I don't get it.

Reply:
I change this